manifests/p/ppy/osu/2024.412.1/ppy.osu.installer.yaml

```diff
@@ -0,0 +1,26 @@
+# Created with komac v2.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: ppy.osu
+PackageVersion: 2024.412.1
+InstallerType: exe
+InstallModes:
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Silent: --silent
+UpgradeBehavior: deny
+ReleaseDate: 2024-04-13
+Scope: user
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/ppy/osu/releases/download/2024.412.1/install.exe
+  InstallerSha256: EEF66D09A4C2B78539A00B1C690027E2020DFFA7B4947F094BEC7231DBB8127B
+Protocols:
+- application/x-osu-beatmap
+- application/x-osu-beatmap-archive
+- application/x-osu-skin-archive
+- application/x-osu-storyboard
+- application/x-osu-replay
```

Comment:
I think protocols refers to eg. `https://`. These are media types, so they should be removed.

osu! currently supports `osu://` and `osump://`. Example: `osu://s/1`.

Here's the list of file types and protocols (URIs): https://github.com/ppy/osu/blob/a780abb0b4b14d539f1e70546cf83798fd167313/osu.Desktop/Windows/WindowsAssociationManager.cs#L41-L53

Reply:
```suggestion
Protocols:
- osu
- osump
FileExtensions:
- osz
- olz
- osr
- osk
```